.travis.yml

```diff
@@ -40,6 +40,7 @@ matrix:
 
 script:
   - cargo test --no-default-features --features="$FEATURES"
```

Comment:
maybe you can consider: `cargo test --all-targets` then examples will be covered.

Reply:
we actually don't need `--all-targets` at all - the default test profile compiles the example files too during it's build step.

tested locally by throwing invalid syntax inside the file, and watching it break.
pushed cleaned up commits

Reply:
Alright :)